### PR TITLE
Userposts

### PR DESCRIPTION
**Pull request by Zoneam**

### CODE REVIEW

File: client/src/components/post/EditPost.js [hunk 1]
```diff
@@ -26,30 +28,48 @@ const EditPostContainer = styled.div`
 `
 
 function EditPost() {
-    const [category, setCategory] = useState('');
-    const [label, setLabel] = useState('');
-    const [title, setTitle] = useState('');
-    const [body, setBody] = useState(''); 
+    const { postId } = useParams();
+    console.log(postId)
```
Comment (vasylynash) [inline client/src/components/post/EditPost.js]: Remove

File: server/schemas/resolvers/posts.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ module.exports = {
       }
     },
     post: async (_, args) => {
+      console.log('args',args)
```
Comment (vasylynash) [inline server/schemas/resolvers/posts.js]: Remove

File: server/schemas/resolvers/posts.js [hunk 3]
```diff
@@ -108,12 +109,13 @@ module.exports = {
       const post = await Post.create({title: title, body: body, category: category, label: label, author: context.user._id });
       await User.findByIdAndUpdate(
                 { _id: context.user._id },
-                { $push: { posts: post._id } },
+                { $push: { posts: post } },
                 { new: true }
               );
       return await post.populate('author').execPopulate();
```
Comment (vasylynash) [inline server/schemas/resolvers/posts.js:114]: await is not needed here

File: server/schemas/resolvers/posts.js [hunk 4]
```diff
@@ -108,12 +109,13 @@ module.exports = {
...
     },
     updatePost: async (_, { id, title, body, category, label }) => {
+      console.log("title",title)
```
Comment (vasylynash) [inline server/schemas/resolvers/posts.js:117]: Remove

**Review by vasylynash — CHANGES_REQUESTED** [review]

**Review by vasylynash — APPROVED** [review]

**Review by Dimavich — APPROVED** [review]

**Review by melinamboedecker — APPROVED** [review]

**Pull request merged**